src/natcap/invest/seasonal_water_yield/seasonal_water_yield_core.pyx

```diff
@@ -441,19 +441,23 @@ cpdef calculate_local_recharge(
     raster_x_size, raster_y_size = flow_dir_raster_info['raster_size']
     cdef _ManagedRaster flow_raster = _ManagedRaster(flow_dir_mfd_path, 1, 0)
 
+    # make sure that user input nodata values are defined
+    # set to -1 if not defined
+    # precipitation and evapotranspiration data should 
+    # always be non-negative
     et0_m_raster_list = []
     et0_m_nodata_list = []
     for et0_path in et0_path_list:
         et0_m_raster_list.append(_ManagedRaster(et0_path, 1, 0))
         et0_m_nodata_list.append(
-            pygeoprocessing.get_raster_info(et0_path)['nodata'][0])
+            pygeoprocessing.get_raster_info(et0_path)['nodata'][0] or -1)
```

Comment:
Neat!  this is a bit of python syntax that I was not expecting to work, but it absolutely does.

Reply:
Oh heck! This is going to have the same problem with truthiness of 0... so `0 or -1` will be `-1`. I can fix this in my next PR...

Reply:
Whoops, so it does!  I'm sorry I didn't catch that.  Yes, let's do this in the next PR.